cscs-checks/apps/lammps/lammps_check.py

```diff
@@ -14,14 +14,19 @@ def __init__(self, name, **kwargs):
         # Reset sources dir relative to the SCS apps prefix
         self.sourcesdir = os.path.join(self.current_system.resourcesdir,
                                        'LAMMPS')
-        self.sanity_patterns = sn.assert_found(r'Total wall time:',
-                                               self.stdout)
-
+        self.reference_energy = {
+            'daint:gpu': -4.6197356,
+            'daint:mc': -4.6195837,
+            'dom:gpu': -4.6189305,
+            'dom:mc': -4.619577
+        }
+        self.energy = sn.extractsingle(r'\s+500000(\s+\S+){3}\s+'
+                                       r'(?P<energy>\S+)\s+\S+\s\n',
```

Comment:
You could move the whole regex string to the next line and it would fit nicely in a single line.